meta_capstone/ViewControllers/InGameViewController.m

```diff
@@ -527,9 +579,10 @@ - (void) createBoard: (NSArray *)words {
     NSString *third = [thirdOptions objectAtIndex:randomIndex];
     NSArray *thirdStartXLocations = [thirdOptionsLocations objectForKey:third];
     [usableWords removeObject:third];
-    [self createTilesTesting:third :  [[thirdStartXLocations objectAtIndex:arc4random() % thirdStartXLocations.count] intValue]:(int) thirdStartY :YES];
+    [self createTiles:third :  [[thirdStartXLocations objectAtIndex:arc4random() % thirdStartXLocations.count] intValue]:(int) thirdStartY :YES];
```

Comment:
by third, do you mean 3rd word? If so, please name more descriptively. Additionally, to make this scale I would suggest passing in a value for how many words you want to add, in this case you can start with 3. You would then loop through the amount of words, and do these location with generic name of tile location etc.

Reply:
fixed naming, will work on scaling as a later stretch goal